### ISSUE TITLE
Video player stars if 

### ISSUE THREAD

**Issue by gigiboeru** [TeamNewPipe/NewPipe#5232]
Hi there,

I'm using the latest version - 20.6 on two different phones.

I've noticed that when I enable autoplay (WiFi or always) whenever I click on a title it ignores the preferred open action and always plays the video (my setting is to start background/audio player).

Is this intended to work like this? Even if so, I consider the preferred open action to be applied on every circumstance of "autoplay".

Thank you,

**Comment by Tisckkytt** [comment]
Yes, I can confirm, preferred to open content in background doesn't work. It would have made one click audio-only playback possible but now we have to long press & manually make the contents play in background.
